CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+# Contributing to Extension Activity Monitor
+
+Firstly, thanks for your interest in contributing to Extension Activity Monitor. Your contribution will make this project more awesome. 🚀
+
+## Table of Contents
+
+- [Project Architecture](https://github.com/atiqueahmedziad/extension-activity-monitor/blob/devdoc/CONTRIBUTING.md#project-architecture)
```

Comment:
Instead of hard-coding a specific path, have you tried to use relative URLs? Like this:
`[Project Architecture](#project-architecture)`

Reply:
Yeah done :-)